Create 2020-01-08-paperMetaproteomics.software.md (#398)
usegalaxy-eu/website: _posts/2020-01-08-paperMetaproteomics.software.md

---
site: freiburg
title: "Survey of metaproteomics software tools for functional microbiome analysis"
tags: [paper]
doi: "10.1101/2020.01.07.897561"
supporters:
 - denbi
 - elixir
---

> Our Galaxy communties of metaproteomics and microbiomics are constantly growing. This article on bioRxiv may be interesting for many of you ["Survey of metaproteomics software tools for functional microbiome analysis"](https://www.biorxiv.org/content/10.1101/2020.01.07.897561v1). Herein, Ray and colleagues are comparing six open software tools for microbiome analysis. From them, Unipept 4.0 and eggNOG-mapper are already installed in the European Galaxy framework.

Reviewer: ```suggestion
Our metaproteomics and microbiomics communities are constantly growing. This article on bioRxiv may be interesting for many of you: ["Survey of metaproteomics software tools for functional microbiome analysis"](https://www.biorxiv.org/content/10.1101/2020.01.07.897561v1). The authors compare six open software tools for microbiome analysis, including Unipept 4.0 and eggNOG-mapper, which are already installed in the European Galaxy framework.
```
